feat: Adapt the document to the release and add JSON views doc (#127)
lenra-io/docs: src/markdown/getting-started/create-project.md

    [{:.btn.link}Read the specific guide for that](../guides/create-from-scratch.html)

## Start developing

Now that you have a new Lenra app project, you can start developing it.

> Be shure you've understand the [Lenra principles](./principles.html) before starting since Lenra is a bit different of the classical frameworks.

Reviewer: ```suggestion
Make sure you've understood the [Lenra principles](./principles.html) before starting since Lenra is a bit different from classic frameworks.
```

### Features

> One of most important thing to know is that Lenra is how to [manage data](../features/data-management.html).

Reviewer: ```suggestion
One of the most important thing to know with Lenra is how to [manage data](../features/data-management.html).
```
